### ISSUE TITLE
Issues trying use the API

### ISSUE THREAD

**Issue by sarry20** [GeorgH93/MarriageMaster#207]
### Information
**Environment information**
*Plugin + server version info*:

```

Marriage Master: 2.4.1-Release
Server: git-Paper-779 (MC: 1.16.5)
Java: 1.8.0_292. 

```

*Online mode*: yes
*BungeeCord*: no

**Server/crash log**

```

<!-- Replace this with your crash log / link to your log file -->
![image](https://user-images.githubusercontent.com/87086679/125125105-70454b00-e0f9-11eb-9942-d7bf4c338d64.png)

```

**Plugin config (optional)**

```


```

### Details
**Description**  
When I try use the class MarriagePlayer it throws this error

**Steps to reproduce**  
put in pom.xml the repo and the dependency, use the method that you provides on instalation process and try use the MarriagePlayer class

**Expected behavior**  
some error in the code

**Other information** (e.g. detailed explanation, related issues, suggestions how to fix, links for us to have context, screenshots, etc.)

**Comment by GeorgH93** [comment]
Please use 2.3.1 as API version for now, the API has not changed since.
The class was removed with 2.4 because of changes made to spigot.

**Comment by sarry20** [comment]
So cant do compatibility with 1.17?

**Comment by GeorgH93** [comment]
It will still work with 1.17.
You can use API version 2.3.1 with the 2.4.1 plugin version.

**Comment by sarry20** [comment]
Ok thanks!

**Comment by GeorgH93** [comment]
Should be fixed with [v2.4.2](https://github.com/GeorgH93/MarriageMaster/releases/tag/v2.4.2).
